### ISSUE TITLE
a small bug

### ISSUE THREAD

**Issue by stl200101** [matanki-saito/CK2dll#139]
When I want to search an area,windows will tip that the game not respond.I need to wait for a long time and can't do anything .

**Comment by matanki-saito** [comment]
Yes, I've been aware of that problem. And it's hard to solve this problem...

**Comment by stl200101** [comment]
I remember that in the old version it isn't a problem.

**Comment by matanki-saito** [comment]
Really ? I will reconfirm that. thx

**Comment by hcccccz** [comment]
Hi, I have same problem when I try to input text through an input method in any version from 3.2.1 to 3.3.3. This problem makes it impossible to search anything in game.

**Comment by stl200101** [comment]
> 真的吗 我将再次确认。谢谢

I have check 4.26version,in this version it can work.
